clearwater-sip-stress-coreonly.root/usr/share/clearwater/sip-stress/callee.xml

```diff
@@ -19,6 +19,58 @@
 
   <send>
     <![CDATA[
+      SIP/2.0 302 Moved Temporarily
+      [last_Call-ID:]
+      [last_CSeq:]
+      [last_From:]
+      [last_To:];tag=[pid]SIPpTag01[call_number]
+      [last_Via:]
+      Server: SIP/2.0
+      RSeq: 462555543
```

Comment:
I don't think you need the RSeq or Require header - I think they're just for 1xx responses and are unrealistic on a 302.